tests/test_flat.py

```diff
@@ -0,0 +1,224 @@
+# This file is part of ci_cpp.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+import os
+import hashlib
+import numpy as np
+import unittest
+
+import lsst.afw.math as afwMath
+import lsst.daf.persistence as dafPersist
+import lsst.ip.isr as ipIsr
+import lsst.meas.algorithms as measAlg
+import lsst.utils.tests
+
+from lsst.pipe.tasks.repair import RepairTask
+from lsst.utils import getPackageDir
+
+
+class DarkTestCases(lsst.utils.tests.TestCase):
```

Comment:
It seems that you copied the tests in "test_dark.py" here into "test_flat.py". However, there are still some names that should be changed (for example, this class is called "DarkTestCases"). also, the docstrings reference the tests in section 5 of DMTN-101 which are for darks; perhaps there should be an explanation that his is the case.

Reply:
No flat tests are defined in DMTN-101.  I thought that I had converted this file, but it appears I did not.  I have switched the test exposure to a flat, so the class will generate a flat-flattened-flat.  We can check the mean and sigma values are plausible.  I've deleted the other tests that are read-noise based (and so not applicable to the flat).